README.md

```diff
@@ -148,7 +148,7 @@ The mapper constructed in the example in the section above could be included in
 
 ###Importing Safely
 
-Vokoder offers many ways to get data into Core Data. The simplest and most approachable interface is offered through the VOKManagedObjectAdditions category. Given an array of dictionaries Vokoder will create or edit managed objects on a background queue and then safely return managed objects to the main queue through the completion block.
+Vokoder offers many ways to get data into Core Data. The simplest and most approachable interface is offered through the VOKManagedObjectAdditions category. Given an array of dictionaries, Vokoder will create or update managed objects on a temporary context and then safely return managed objects from the main context through a completion block on the main queue.
```

Comment:
Looking at the implementation of `vok_addWithArrayInBackground:completion:`, I'm wondering if it should use `performBlockAndWait:`?